pkg/aws/policies.go

```diff
@@ -926,6 +968,16 @@ func (c *awsClient) detachOperatorRolePolicies(role *string) error {
 	return nil
 }
 
+/*func (c *awsClient) deleteOperatorRolePolicies(policies []*iam.AttachedPolicy) error {
+	for _, policy := range policies {
+		_, err := c.iamClient.DeletePolicy(&iam.DeletePolicyInput{PolicyArn: policy.PolicyArn})
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}*/
```

Comment:
Please remove commented out code (it will all be in the git history anyway, so no need to keep this stuff around).